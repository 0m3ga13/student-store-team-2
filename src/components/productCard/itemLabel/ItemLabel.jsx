"use client";
import { useTranslations } from "next-intl";
import React from "react";

const ItemLabel = ({ textContent }) => {
    const t = useTranslations("Index");

    let style = "";
    if (textContent === t("For Sale")) {
        style = `bg-secondaryBlue`;
    } else if (textContent === t("For Borrow")) {
        style = `bg-secondaryGreen`;
    } else {
        style = `bg-secondaryGreen`;
    }

    return (
        <div
            className={`flex justify-center text-center items-center font-jost font-bold rounded-full ${style} text-white text-sm w-[105px] h-[26px]`}
        >
            {textContent}
        </div>
    );
};

export default ItemLabel;
