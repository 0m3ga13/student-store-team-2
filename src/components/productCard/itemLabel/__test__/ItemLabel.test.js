import renderer from "react-test-renderer";

import ItemLabel from "../ItemLabel";

it("renders correctly", () => {
    const tree = renderer.create(<ItemLabel />).toJSON();
    expect(tree).toMatchSnapshot();
});
