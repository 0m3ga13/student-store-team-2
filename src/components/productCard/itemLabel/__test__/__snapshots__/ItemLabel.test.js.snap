// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex justify-center text-center items-center font-jua rounded-full bg-secondaryGreen text-white text-sm w-[105px] h-[26px] subpixel-antialiased"
/>
`;
